src/cdk/overlay/overlay-ref.ts

```diff
@@ -268,6 +305,11 @@ export class OverlayRef implements PortalOutlet, OverlayReference {
     return this._keydownEventsObservable;
   }
 
+  /** Gets an observable of mouse outside click events targeted to this overlay. */
+  clickEvents(): Observable<MouseEvent> {
```

Comment:
Is this clicks _inside_ or _outside_ the overlay?

Reply:
definitely `outsideClickEvents`